JustSaying.UnitTests/AwsTools/MessageHandling/Sns/TopicByName/WhenPublishingAsyncExceptionCanBeHandled.cs

```diff
@@ -23,7 +23,7 @@ public class WhenPublishingAsyncExceptionCanBeHandled : XAsyncBehaviourTest<SnsT
 
         protected override SnsTopicByName CreateSystemUnderTest()
         {
-            var topic = new SnsTopicByName("TopicName", _sns, _serialisationRegister, Substitute.For<ILoggerFactory>(), new SnsWriteConfiguration
+            var topic = new SnsTopicByName("TopicName", _sns, _serialisationRegister, _ => {}, Substitute.For<ILoggerFactory>(), new SnsWriteConfiguration
```

Comment:
As Martin mentioned in his comment, happy to move this into a static `NoOp` property for clarity.